### PR TITLE
Update user

### LINKED ISSUE #5: Create Mutation for updateUser
- [ ] Update Pitch

- [ ] Update Speed

- [ ] Voice

### LINKED ISSUE #35: Create Mutation for Update User Speed and Voice

### PR DESCRIPTION
**Pull request by cgaddis36**
### Description
Add mutation for updating a User

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [x ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: spec/requests/v1/section_query_spec.rb [hunk 1]
```diff
@@ -26,11 +26,4 @@
       expect(parsed_response[:data][:sections].first[:title]).to eq('greetings')
       expect(parsed_response[:data][:sections].first[:icon]).to eq('red')
   end
```
Comment (cgaddis36) [inline spec/requests/v1/section_query_spec.rb:28]: just a small cleanup while browsing spec files

File: spec/requests/v1/user_update_mutation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+      post "/v1/graphql", params: { query: mutation_string }
+
+      expect(response.status).to eq(200)
+      parsed_response = JSON.parse(response.body, symbolize_names: true)
+
+      expect(parsed_response[:data][:updateUser][:user][:id]).to eq(bob.id.to_s)
+      expect(parsed_response[:data][:updateUser][:user][:voice]).to eq("Rebecca")
+      expect(parsed_response[:data][:updateUser][:user][:speed]).to eq(0)
+  end
```
Comment (cgaddis36) [inline spec/requests/v1/user_update_mutation_spec.rb:39]: checking the attributes are correctly updated in the DB

File: app/graphql/mutations/users/update_user.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+module Mutations
+  module Users
+    class UpdateUser < ::Mutations::BaseMutation
+      argument :id, ID, required: true
```
Comment (sagemlee) [inline app/graphql/mutations/users/update_user.rb:5]: nice, this looks really good.

**Review by sagemlee — APPROVED** [review]

File: app/graphql/mutations/users/update_user.rb (hunk 3 above)
Comment (EricLarson2020) [reply app/graphql/mutations/users/update_user.rb:5]: Nice work here

File: spec/requests/v1/user_update_mutation_spec.rb (hunk 2 above)
Comment (EricLarson2020) [reply spec/requests/v1/user_update_mutation_spec.rb:39]: good work with the test!

**Review by EricLarson2020 — APPROVED** [review]

**Pull request merged**